libs/template/let/src/lib/docs/Readme.md

```diff
@@ -0,0 +1,496 @@
+| `renderCallback` | `Subject<U>`                                                       | giving the developer the exact timing when the `LetDirective` created, updated, removed its template. Useful for situations where you need to know when rendering is done.                              |
+| `strategy`       | `Observable<RxStrategyNames \ string> \ RxStrategyNames \ string>` | _default: `normal`_ configure the `RxStrategyRenderStrategy` used to detect changes                                                                                                                     |
+
+
+**Outputs**
+
+n/a
+
+**Performance Features**
+
+- value binding is always present. ('`*ngIf` hack' bugs and edge cases)
+- lazy template creation (done by render strategies) 
+- triggers change-detection on `EmbeddedView` level
+- distinct same values in a row (over-rendering)
+- view memoization
+
+# Setup
+
+The `LetModule` can be imported as following:
+
+Module based setup:
+```
+import { LetModule } from "@rx-angular/template/let";
+
+@NgModule({
+  imports: [ LetModule ],
+  // ...
+})
+export class AnyModule {}
+```
+
+Standalone component setup:
+```
+import { LetModule } from "@rx-angular/template/let";
+
+@NgComponent({
+    standalone: true,
+    imports: [ LetModule ],
+    template: `...`
+})
+export class AnyComponent {}
+```
+
+# Basic Usage
+
+> **⚠ Notice:**  
+> By default `*rxLet` is optimized for performance out of the box.
+> 
+> This includes:
+> - The default render strategy is [`normal`](https://github.com/rx-angular/rx-angular/blob/main/libs/cdk/render-stractgies/src/docs/README.md). This ensures non-blocking rendering. See [strategy configuration](https://github.com/rx-angular/rx-angular/blob/main/libs/cdk/render-stractgies/src/docs/README.md#Default-configuration) if you want to change it. 
+> - Runs the `EmbeddedView` creation outside of `NgZone`. See [how to opt-in to `NgZone`](https://github.com/rx-angular/rx-angular/blob/main/libs/cdk/render-stractgies/src/docs/README.md) if you want to change it.
+> - Creates templates lazy and manages multiple template instances
+> 
+
+## Binding an Observable to a local variable in the template
+
+The `*rxLet` directive makes it easy to work with reactive data streams in the template.
+This can be achieved by using Angular's native 'let' syntax `*rxLet="observableNumber$; let n"`.
+
+```html
+<ng-container *rxLet="observableNumber$; let n">
+  <app-number [number]="n"></app-number>
+  <app-number-special [number]="n"></app-number-special>
+</ng-container>
+```
+
+## Using the reactive context
+
+![Contextual-State--template-vs-variable](https://user-images.githubusercontent.com/10064416/192660150-643c4d37-5326-4ba2-ad84-e079890b3f2f.png)
+
+A nice feature of the `*rxLet` directive is, it provides 2 ways to access the [reactive context state]() in the tempalte:
```

Comment:
missing link to reactive context state -> https://github.com/rx-angular/rx-angular/blob/a6f321941e1767ae0809b2e0bb01fa98cdd8647f/libs/template/docs/reactive-context.md 

typo `tempalte` -> `template`